### ISSUE TITLE
bug:  EOF while parsing a value at line 1 column 0

### ISSUE THREAD

**Issue by lucasazevedonet** [withfig/fig#2056]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 13.0.1 (22A400)

### Expected behaviour

![image](https://user-images.githubusercontent.com/2625934/202013684-5dacb5b8-32a3-46f7-85b9-824569841fbd.png)

### Actual behaviour

![image](https://user-images.githubusercontent.com/2625934/202013853-1cdae27e-51bd-484c-9028-48b2692c3b31.png)

### Environment

```yaml
fig-details:
  - 2.7.5
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Apple M1
  - cores: 8
  - mem: 8.00 GB
os-info:
  - macOS 13.0.1 (22A400)
environment:
  - shell: /Applications/Fig.app/Contents/MacOS/fig_desktop
  - terminal: <unknown>
  - cwd: /
  - exe-path: /Applications/Fig.app/Contents/MacOS/fig-darwin-universal
  - install-method: unknown
  - env-vars:
    - PATH: /usr/bin:/bin:/usr/sbin:/sbin
    - SHELL: /bin/zsh
```

**Comment by grant0417** [comment]
Hey there @lucasazevedonet, this was fixed by Fig 2.7.6 I believe (the current version is 2.7.9), update Fig and this should be resolved for you!

**Comment by lucasazevedonet** [comment]
Thank you! You can close the issue.
